src/main/features/dashboard/views/search.njk

```diff
@@ -0,0 +1,25 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import linkButton, csrfProtection, errorSummary, searchInput %}
+
+{% set title = t('Money claim') %}
+{% set heading = t('Civil Money Claims') %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <form novalidate method="post">
+        {{ csrfProtection(csrf) }}
+
+        {{ linkButton('Issue a new claim', 'start', 'button button-start') }}
+
+        <h2 class="heading-medium" style="margin-bottom: 0">Confirm service</h2>
```

Comment:
no inline styles, CSP should block that anyway?

Reply:
didn't even notice this. was not seeing any errors for it.